x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/plugin/RestSqlQueryAction.java

```diff
@@ -130,6 +130,21 @@ public RestResponse buildResponse(SqlQueryResponse response) throws Exception {
         });
     }
 
+    private XContentType getXContentType(String accept) {
+        if (accept == null) {
+            return XContentType.JSON;
+        }
+        XContentType xContentType = XContentType.fromFormat(accept);
+        if (xContentType != null) {
+            return xContentType;
+        }
+        xContentType = XContentType.fromMediaType(accept);
+        if (xContentType != null) {
+            return xContentType;
+        }
+        return xContentType;
```

Comment:
Just a small compacting suggestion: 
```suggestion
        XContentType xContentType = XContentType.fromFormat(accept);
        return xContentType != null ? xContentType : XContentType.fromMediaType(accept);
```